package.json

```diff
@@ -61,6 +61,7 @@
     "karma-mocha": "^1.3.0",
     "karma-ng-html2js-preprocessor": "^1.0.0",
     "karma-spec-reporter": "0.0.32",
+    "locale": "^0.1.0",
```

Comment:
This should be in `api/package.json` in the `dependencies` section, rather than at the root level and in the `devDependencies` section.

Reply:
Thanks!